src/Illuminate/Http/Request.php

```diff
@@ -459,7 +459,7 @@ protected function convertUploadedFiles(array $files)
      *
      * @param  string  $key
      * @param  mixed  $default
-     * @return \Symfony\Component\HttpFoundation\File\UploadedFile|array|null
+     * @return \Illuminate\Http\UploadedFile|array|null
```

Comment:
This is wrong, the files are actually an instance of \Symfony\Component\HttpFoundation\File\UploadedFile